src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -470,9 +563,10 @@ const useGroupInfo = (channelId = null) => {
     newParticipant,
     handleCloseSelectUser,
     onRemoveUser,
+    onAddMember,
     openModal,
     leaveGroup,
-    alertRemoveUser,
+    handleOpenPopup,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return statement of the hook has been updated to include the new functions. This is necessary for the new functionality being added.
```diff
+ onAddMember,
+ leaveGroup,
+ handleOpenPopup,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->